paymentsheet/res/layout/layout_paymentsheet_add_new_payment_method_item.xml

```diff
@@ -18,7 +18,7 @@
             android:id="@+id/plus_icon"
             style="@style/StripePaymentOptionItemCardImage"
             android:src="@drawable/stripe_ic_paymentsheet_add"
-            android:contentDescription="@null" />
+            android:contentDescription="@string/title_add_a_card" />
```

Comment:
It's only card now, but it won't be after GA. It should be more general, like "Add a new payment method".